Inspicio/wwwroot/js/ajaxThumbs.js

```diff
@@ -10,20 +10,23 @@
         className = className.slice(0, 12) + "-o" + className.slice(12);
     }
     if (button === "thumbs-up") {
-        updateThumbs(true, button);
+        updateThumbs(1, button);
```

Comment:
I think it's worth pulling out the 1, 0 & -1 into named constants (not that JS has constants) to make it clear what they mean.

[[++____]](https://gist.github.com/WilkaH/29df175966a76687f846)